docs/source/ruletypes.rst

```diff
@@ -1277,6 +1277,10 @@ evaluated separately against the threshold(s).
 For example, "%.2f" will round it to 2 decimal places.
 See: https://docs.python.org/3.4/library/string.html#format-specification-mini-language
 
+.. note:: If you want to change the percentage format inside your alert, you must specify the format there cause ``percentage_format_string`` wont't be applied.
```

Comment:
Some typos on this new text. I suggest the following instead:

If you want to change the percentage format inside your alert, you must specify the format there because ``percentage_format_string`` won't be applied.

Example: ``alerta_value: "%(percentage).2f%%"``